src/main/java/com/kodilla/ecommercee/OrderController.java

```diff
@@ -1,4 +1,40 @@
 package com.kodilla.ecommercee;
 
+import com.kodilla.ecommercee.domain.OrderDto;
+import org.springframework.web.bind.annotation.*;
+import java.util.*;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@RestController
+@RequestMapping("/v1/orders")
 public class OrderController {
+
+    @RequestMapping(method = RequestMethod.GET, value = "/orders")
```

Comment:
Nie ma sensu podwójnie nazywać tych ścieżek.
Teraz masz adres /v1/orders/orders
Wystarczy samo GET /v1/orders żeby było wiadomo, że chodzi tutaj o pobranie zamówień

Przy okazji spróbuj zapisać to przy użyciu nieco nowszych adnotacji -GetMapping itp
https://www.baeldung.com/spring-new-requestmapping-shortcuts